server/src/main/java/com/lunchtime/controllers/MenuItemDishController.java

```diff
@@ -56,6 +54,21 @@ public MenuItemDishController(MenuItemDishService menuItemDishService) {
         return ResponseEntity.ok(menuItemDishPage);
     }
 
+    @GetMapping("/category")
+    public ResponseEntity<Page<MenuItemDish>> getAllByDishCategoryName(
+        @RequestParam("name") String name,
```

Comment:
Maybe it isn't important, but your API is case sensitive (it returns "Pizza", but doesn't return "pizza")

Reply:
I think we don't need it here .  On client part user choose between existing categories and don't type it manually.